.github/dependabot.yml

```diff
@@ -105,3 +105,13 @@ updates:
       day: "wednesday"
     open-pull-requests-limit: 4
     rebase-strategy: "disabled"
+  - package-ecosystem: npm
+    directory: "/bridge"
+    schedule:
+      interval: weekly
+      day: "wednesday"
+  - package-ecosystem: npm
+    directory: "/bridge/server"
+    schedule:
+      interval: weekly
+      day: "wednesday"
```

Comment:
Is this necessary, or is it already covered by the configuration for directory `"/bridge"`?

Reply:
Subfolders that have their own package.json need to be included, so that's required.